### PR TITLE
Creates polymorphic housing media links

### PR DESCRIPTION
**Pull request by kwiliarty**

### CODE REVIEW

**Comment by kwiliarty** [comment]
Reviewing this myself I think I can and should just take out the heading and subheading pieces. We needed that for the building attributes because they were more complicated.

**Comment by kwiliarty** [comment]
Need to add the links on the building show page

**Comment by kwiliarty** [comment]
Open in new tab

**Comment by kwiliarty** [comment]
Add building and unit links on the match page, fix building edits

**Comment by kwiliarty** [comment]
Hard code heading, but make it translatable.

**Comment by kwiliarty** [comment]
Show url in edit table

File: app/views/housing_media_links/new.haml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+-# Partial: used polymorphically for building and unit attributes
+-#   Arguments:
+-#     housingable: currently a building or unit object
+-#     link: a HousingMediaLink object
+
+= content_for :modal_title do
+  Add Photo or Video Link
```
Comment (eanders) [inline app/views/housing_media_links/new.haml]: Let's translate this `= _('Add Photo or Video Link')`

**Comment by kwiliarty** [comment]
> Need to add the links on the building show page

And actually, it is helpful here to have a parameterized heading after all, because we need to suppress the heading in some contexts, and this is one of them.

**Comment by kwiliarty** [comment]
Adding the links to the match pages was slightly tricky, so that was the only part that wasn't fast.

**Pull request merged**